### PR TITLE
Add token_file_path parameter and enable auth

### PR DESCRIPTION
**Pull request by cschuet**

### CODE REVIEW

File: cartographer/cloud/internal/local_trajectory_uploader.cc [hunk 1]
```diff
@@ -161,7 +172,8 @@ void LocalTrajectoryUploader::AddTrajectory(
   *request.add_expected_sensor_ids() =
       cloud::ToProto(GetLocalSlamResultSensorId(local_trajectory_id));
   async_grpc::Client<handlers::AddTrajectorySignature> client(client_channel_);
-  CHECK(client.Write(request));
+  ::grpc::Status status;
+  CHECK(client.Write(request,& status)) << status.error_message();
```
Comment (gaschler) [inline cartographer/cloud/internal/local_trajectory_uploader.cc]: style: &status

File: cartographer/cloud/internal/local_trajectory_uploader.cc [hunk 2]
```diff
@@ -36,12 +37,14 @@ namespace {
 using common::make_unique;
 
 constexpr int kConnectionTimeoutInSecond = 10;
+constexpr int kTokenRefreshIntervalInMinutes = 1;
```
Comment (gaschler) [inline cartographer/cloud/internal/local_trajectory_uploader.cc]: I prefer SI units like seconds, but I leave this to you.

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Comment by wally-the-cartographer** [comment]
Refusing to merge, since the combined checks state is failure

**Pull request merged**